xchange-itbit/src/main/java/org/knowm/xchange/itbit/v1/ItBitAdapters.java

```diff
@@ -69,7 +68,7 @@ private static DecimalFormat getCryptoFormat() {
     cryptoFormat.setDecimalFormatSymbols(CUSTOM_SYMBOLS);
     cryptoFormat.setMaximumFractionDigits(4);
     cryptoFormat.setGroupingUsed(false);
-    cryptoFormat.setRoundingMode(RoundingMode.HALF_UP);
+    cryptoFormat.setRoundingMode(RoundingMode.UNNECESSARY);
```

Comment:
-1
this will cause ArithmeticException's
https://docs.oracle.com/javase/7/docs/api/java/math/RoundingMode.html

Reply:
ok, but this will cause 0.001000034 to also throw an exception
don't you think that a `HALF_DOWN` and a `if(someDecimal.compareTo(MIN_AMOUNT) < 0) throw new ItBitException` would be better then relaying on the formatter for validation?

Reply:
Truncating extra precision